### ISSUE TITLE
Please help me to configure Linked-Data-Theater virtuoso part with easy step or with any example.

### ISSUE THREAD

**Issue by ifourvimal** [architolk/Linked-Data-Theatre#68]
Hi,

Is anybody please help me to setup Linked-Data-Theater. I am a beginner in this stuff and I need easy and step by step procedure to do so that I can run the Linked-Data-Theater.

I have first installed all the prerequisite from below link
https://github.com/architolk/Linked-Data-Theatre/wiki/Theatre-installation

I have downloaded the "ldt-1.20.0.war" and imported to my eclipse project.

I have tried to load the virtuoso graph from the URL below,
https://github.com/architolk/Linked-Data-Theatre/wiki/Theatre-configuration

and I have taken link from that example
http://dbpedia.org/resource/Amersfoort as graph IRI as like this http://localhost:8080/stage but 

When I run the project I am getting an error 404 not found which step I missed can anybody help me on this.

Thank you in advance.

**Comment by architolk** [comment]
Do you have a working version of a Virtuoso triplestore?

**Comment by ifourvimal** [comment]
Yes, I have a version of a Virtuoso triple store which runs smoothly on my computer.I am also accessing this through the URL http://localhost:8890/ .

**Comment by architolk** [comment]
Ok, maybe running it from eclipse might create some problems. Important is that the domain plus port is the same for the stage as for the URL. Some checks:
- Go to http://localhost:8080/info. You should see a page with some info regarding your configuration.
- Make sure that the value for "Public backstage" is "yes". This is the default setting (look at the comments in config.xml to change this, or look at SECURE.md for more information).
- Go to http://localhost:8080/backstage. You might need to enter some credentials (the default war creates a secure context for a user with role backstageadmin), in a default Tomcat installation, you would set these in tomcat-users.xml, see also SECURE.md)
- You should see a list of available representations.

**Comment by architolk** [comment]
Also, you could try to upload the basic configuration: `curl -X PUT -T basic-configuration.ttl http://localhost:8890/sparql-graph-crud?graph-uri=http://localhost:8080/stage`

**Comment by architolk** [comment]
This basic configuration works at http://localhost:8080, so you might go from that point.
